dashboard/src/pages/CreateEventForm.vue

```diff
@@ -4,84 +4,120 @@
+              description="The event link will be as shown above."
+            />
+            <FormControl
+              v-model="temp_event.status"
+              :type="'select'"
+              :options="[
+                {
+                  label: 'Draft',
+                  value: 'Draft',
+                },
+                {
+                  label: 'Live',
+                  value: 'Live',
+                },
+                {
+                  label: 'Concluded',
+                  value: 'Concluded',
+                },
+                {
+                  label: 'Cancelled',
+                  value: 'Cancelled',
+                },
+              ]"
+              size="md"
+              label="Event Status&ast;"
+              description="Current status of the event."
+            />
+            <TextEditor
+              label="Event Description&ast;"
+              class="col-span-2"
+              placeholder="Write an event description"
+              :model-value="temp_event.event_description"
+              @update:model-value="($event) => (temp_event.event_description = $event)"
+            />
+          </div>
         </div>
-        <div class="font-semibold text-gray-800 border-b-2 pb-2">Location</div>
-        <div class="p-2 my-1 grid sm:grid-cols-1 md:grid-cols-2 gap-x-4 gap-y-6">
-          <FormControl v-model="temp_event.location" :type="'text'" size="md" label="Location" />
-          <FormControl v-model="temp_event.map_link" :type="'url'" label="Map Link" side="md" />
+        <div class="space-y-2">
+          <div class="prose">
+            <h3>Location</h3>
+          </div>
+          <div class="grid sm:grid-cols-1 md:grid-cols-2 gap-x-4 gap-y-6">
+            <FormControl
+              v-model="temp_event.location"
+              :type="'text'"
+              size="md"
+              label="Location&ast;"
+            />
+            <FormControl v-model="temp_event.map_link" :type="'url'" size="md" label="Map Link" />
+          </div>
         </div>
-        <div class="font-semibold text-gray-800 border-b-2 pb-2">Timeline</div>
-        <div class="p-2 my-1 grid sm:grid-cols-1 md:grid-cols-2 gap-x-4 gap-y-6">
-          <FormControl
-            v-model="temp_event.event_start_date"
-            :type="'datetime-local'"
-            size="md"
-            label="Event Start Date"
-          />
-          <FormControl
-            v-model="temp_event.event_end_date"
-            :type="'datetime-local'"
+        <div class="space-y-2">
+          <div class="prose">
+            <h3>Timeline</h3>
+          </div>
+          <div class="grid sm:grid-cols-1 md:grid-cols-2 gap-x-4 gap-y-6">
+            <FormControl
+              v-model="temp_event.event_start_date"
+              :type="'datetime-local'"
+              size="md"
+              label="Event Start Date&ast;"
+            />
+            <FormControl
+              v-model="temp_event.event_end_date"
```

Comment:
Here would be good to add a validation to ensure that end_date > start_date always